<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1040</width>
      <height>585</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
	border-image:url(:/res/background.png);
}

QLabel {
	front: 25 10pt &quot;微软雅黑&quot;;
	border-radius: 4px;
	background-color: rgba(60, 60, 60, 100);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>931</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEditCity"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>380</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="searchBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/search.png</normaloff>:/res/search.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="dateLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>421</width>
       <height>121</height>
      </rect>
     </property>
     <widget class="QLabel" name="lblTypeIcon">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>19</y>
        <width>311</width>
        <height>111</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_8" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>231</width>
         <height>45</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblTypeToday">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblHighLow">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_9" native="true">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>3</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
      <widget class="QLabel" name="lblTemperature">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>110</width>
         <height>68</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="addressLabel">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>135</width>
         <height>59</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>401</width>
       <height>291</height>
      </rect>
     </property>
     <widget class="QLabel" name="coldIndexLabel">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_10" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>59</y>
        <width>401</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 85, 255);
background-color: rgb(255, 170, 255);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_11" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lblwindIconToday">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_32" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="lblFxToday">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblFlToday">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
         <zorder>widget_32</zorder>
         <zorder>lblwindIconToday</zorder>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_12" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lblPM25Icon">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_39" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_27">
             <item>
              <widget class="QLabel" name="windIcon_6">
               <property name="text">
                <string>PM2.5</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblPM25">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
         <zorder>widget_39</zorder>
         <zorder>lblPM25Icon</zorder>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_13" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="lblHumidityIcon">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_46" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_28">
             <item>
              <widget class="QLabel" name="windIcon_9">
               <property name="text">
                <string>湿度</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblHumidity">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget_14" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="aqiIconToday">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_47" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_29">
             <item>
              <widget class="QLabel" name="windIcon_12">
               <property name="text">
                <string>空气质量</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblAqiToday">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>471</width>
       <height>89</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>240</y>
       <width>541</width>
       <height>351</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_19" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>521</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_23" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="lblHighCurve">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_24" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="lblLowCurve">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_18" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>511</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_26" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="lblFx0">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblFl0">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_27" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="lblFx1">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblFl1">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_28" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="lblFx2">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblFl2">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_29" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="lblFx3">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblFl3">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_30" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="lblFx4">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblFl4">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_31" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QLabel" name="lblFx5">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblFl5">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_16" native="true">
     <property name="geometry">
      <rect>
       <x>459</x>
       <y>49</y>
       <width>541</width>
       <height>221</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_15" native="true">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>0</y>
        <width>471</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(64, 223, 255);
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <widget class="QLabel" name="lbDate5">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>11</y>
         <width>69</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblWeek3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>34</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblWeek5">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>34</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbDate0">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>11</y>
         <width>69</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblWeek2">
       <property name="geometry">
        <rect>
         <x>161</x>
         <y>34</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbDate1">
       <property name="geometry">
        <rect>
         <x>87</x>
         <y>11</y>
         <width>69</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblWeek1">
       <property name="geometry">
        <rect>
         <x>85</x>
         <y>34</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbDate3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>11</y>
         <width>69</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblWeek4">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>34</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbDate4">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>11</y>
         <width>69</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblWeek0">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>34</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbDate2">
       <property name="geometry">
        <rect>
         <x>163</x>
         <y>11</y>
         <width>69</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_17" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="5">
        <widget class="QLabel" name="lblQuality5">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="lblQuality4">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="lblQuality3">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lblQuality2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lblQuality1">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblQuality0">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_20" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>491</width>
        <height>101</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_21" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>481</width>
         <height>71</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblTypeIcon0">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>71</width>
          <height>69</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lblTypeIcon1">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>0</y>
          <width>71</width>
          <height>69</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lblTypeIcon2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>0</y>
          <width>71</width>
          <height>69</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lblTypeIcon3">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>71</width>
          <height>69</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lblTypeIcon4">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>0</y>
          <width>71</width>
          <height>69</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lblTypeIcon5">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>0</y>
          <width>71</width>
          <height>69</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_22" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblType0">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblType1">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblType2">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblType3">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblType4">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblType5">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
